AppBuilder/platform/views/viewComponent/ABViewGridComponent.js

```diff
@@ -1371,13 +1371,34 @@ export default class ABViewGridComponent extends ABViewComponent {
 
       if (state.value !== state.old) {
          const item = $DataTable?.getItem(editor.row);
+         const CurrentObject = this.datacollection.datasource;
+         const abWebix = this.AB.Webix;
 
          item[editor.column] = state.value;
 
          $DataTable.removeCellCss(item.id, editor.column, "webix_invalid");
          $DataTable.removeCellCss(item.id, editor.column, "webix_invalid_cell");
 
-         const CurrentObject = this.datacollection.datasource;
+         //maxlength field
+         const f = CurrentObject.fieldByID(editor.config.fieldID);
+         if (
+            f.settings.maxLength &&
+            state.value.length > f.settings.maxLength
+         ) {
+            abWebix.alert({
```

Comment:
we can use [`this.AB.alert()`](https://github.com/digi-serve/ab_platform_web/blob/1ec21422c5edca89e8b98feb6ec7cb6bc6abb9a0/AppBuilder/ABFactory.js#L839) here. It just wraps `webix.alert()`, but it could give flexibility if we ant to change `AB.alert` in the future

Reply:
Thank you very much.